src/board.js

```diff
@@ -0,0 +1,73 @@
+import { useState } from 'react'
+import { Square } from './square.js'
+
+const board = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9]
+
+export function Board(props) {
+  const [squareColors, setSquareColors] = useState(Array(board.length * board.length).fill('grey'))
+  const [clickedSquare, setClickedSquare] = useState(null)
+
+  const handleSquareClick = (i) => {
+    const squares_slice = squareColors.slice()
+    squares_slice[i] = 'red'
```

Comment:
pull out the string 'red' into constants file so you dont have to change in multiple places when you wanna update color

Reply:
Haha ya it's got lots of dumb stuff like this because it's quick and dirty right now 😅